docs/plugins/caching/how-to.mdx

```diff
@@ -1,17 +1,23 @@
 ---
-sidebar_position: 4
-sidebar_label: Caching
-description: Explore the caching plugin, and how to incorporate it into your Hasura projects.
+title: How to Configure the Caching Plugin
+sidebar_position: 2
+description: "Learn how to configure the Caching Plugin for Hasura DDN."
 keywords:
   - hasura plugins
-  - plugins
+  - caching plugin
+  - plugins architecture
   - engine plugins
-  - custom execution
-  - caching
-seoFrontMatterUpdated: false
+  - how-to
+  - guide
 ---
 
-# Server-side Caching
+# How to Configure the Caching Plugin
+
+## Introduction
+
+## Step 1.
```

Comment:
This must be a mistake. There are 3 level 2 headings and nothing in introduction or step 1.

Reply:
Probably "step 1 and "next steps" should be removed here since the following text is an introduction.

Reply:
Great catch; that was from my stubbing early on 🙌